### PR TITLE
Move steps out of CommandLine

### PR DESCRIPTION
**Pull request by phil-davis**
## Description
1) Move ``occ`` command steps into ``OccContext``
2) Move transfer-ownership steps into ``TransferOwnershipContext``
3) Adjust calls and ``behat.yml`` as required.

## Motivation and Context
Work towards moving steps that are in acceptance test trait files (which all get lumped into FeatureContext) into the relevant Context files.

## How Has This Been Tested?
CI

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests

## Checklist:
- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:
- [ ] Backport (if applicable set "backport-request" label and remove when the backport was done)

### CODE REVIEW

**Review by individual-it — APPROVED** [review]

**Comment by individual-it** [comment]
```
    And as "user1" folder "/sub/test" should not exist in the last received transfer folder                # TransferOwnershipContext::asFileOrFolderShouldNotExist()
      Behat\Testwork\Call\Exception\FatalThrowableError: Fatal error: Call to undefined method FeatureContext::getLastTransferPath() in /drone/src/tests/acceptance/features/bootstrap/TransferOwnershipContext.php:130
```

**Comment by phil-davis** [comment]
I need to find calls that I missed.

File: tests/acceptance/features/bootstrap/TransferOwnershipContext.php [hunk 1]
```diff
@@ -84,12 +142,12 @@ public function asFileOrFolderShouldNotExist($user, $entry, $path) {
 	 * @param string $entry
 	 * @param string $path
 	 *
-	 * @return array
+	 * @return void
 	 * @throws \Exception
 	 */
 	public function asFileOrFolderShouldExist($user, $entry, $path) {
-		$path = $this->featureContext->getLastTransferPath() . $path;
-		return $this->featureContext->asFileOrFolderShouldExist(
```
Comment (phil-davis) [inline tests/acceptance/features/bootstrap/TransferOwnershipContext.php]: Nowadays ``featureContext->asFileOrFolderShouldExist`` returns void. Nothing uses the supposed return of this local ``asFileOrFolderShouldExist`` method anyway. So make it ``void`` also.

**Comment by phil-davis** [comment]
Backport ``stable10`` #34128

**Pull request merged**